bigquery_etl/glean_usage/templates/baseline_clients_first_seen_v1.init.sql

```diff
@@ -0,0 +1,39 @@
+{{ header }}
+{% from "macros.sql" import core_clients_first_seen %}
+
+CREATE TABLE IF NOT EXISTS
+  `{{ first_seen_table }}`
+PARTITION BY
+  first_seen_date
+CLUSTER BY
+  sample_id
+OPTIONS
+  (require_partition_filter = FALSE)
```

Comment:
Setting this to `TRUE` breaks testing, which relies on `select * from {{ table }}`. It might not be that important to have a partition filter on these tables anyhow.